BP/common/religion/religions/00_christianity.txt

```diff
@@ -853,11 +849,13 @@
 			icon = mfi_hands_of_god
 			color = { 0.2 0.1 0.0 }
 			
+			#WtWSMS: jerusalem, alexandria, constantinople, rome, antioch
 			holy_site = jerusalem
-			holy_site = esztergom
-			holy_site = rome
-			holy_site = santiago
+			holy_site = alexandria #Arianus' Seat
 			holy_site = constantinople
+			holy_site = nikopolis #Ulfias' wrote the Gothic Bible here
```

Comment:
Arianism competes with Chalcedonian Christianity, so they should have the same holy sites